package.json

```diff
@@ -5,7 +5,8 @@
   "author": "Christian Joudrey",
   "main": "lib/index.js",
   "scripts": {
-    "fmt": "node_modules/prettier/bin/prettier.js --single-quote --trailing-comma es5 --write -l \"{test,src}/**/*.js\"",
+    "fmt": "prettier --single-quote --trailing-comma es5 --write -l \"{test,src}/**/*.js\"",
```

Comment:
Not necessary as `yarn run` sets up the path and uses script in `/node_modules/.bin`